### DISCUSSION TITLE
lede社区交流环境问题

### DISCUSSION THREAD

**Discussion by blzadx** [coolsnowwolf/lede#12010] [Polls / 投票选举]
- 简单来说就是自嘲了一下lede社区的环境，被管理员NICK BOT误解成挑衅，然后在TG大群被BAN了，后续讨说法也被直接无视，无奈只能到GITHUB社区讨公道。虽然确实很气愤，但也不想拼个你死我活，就是想看看这个社区整体到底是什么氛围，好决定自己在社区里的去留。
- 下面截图里面，Y是我，nick bot是管理员。我是觉得他在问“用户为什么来社区”，所以模仿用户的口气说话。其实当时就是隐约感觉有一句很不着边际的反讽台词，很适合拿来自嘲，但一时又没想起来，于是就只能模仿口气说了。然后在编写这个disscussion的时候想起来了，“佛山谁最能打？”，“当然是我啊，难道叶问啊？”（出自几年前的B站鬼畜）
<img title="模仿用户" src="https://github.com/coolsnowwolf/lede/assets/67340056/79751b06-6749-4d85-9676-eb4103abddf6" alt="模仿用户"   style="zoom:25%;">

- 下面是讨说法被无视。其实后面我新创了个号去群里解释了一遍，但也被秒BAN，没来得及截图，不知道他有没有看。我也不想再创新号折腾了。

<img title="讨说法" src="https://github.com/coolsnowwolf/lede/assets/67340056/21a4593c-7ffc-4fb7-8418-499f2302ce52" alt="讨说法"   style="zoom:25%;">

- 关于我自己，其实就是一个业余爱好者，虽然确实没给项目做出什么巨大贡献，但也在用自己掌握的知识帮助有需要的人。群里那个小米UU插件移植教程是我第一个作品，把可以加速PC和安卓的厂商版UU插件给移植到同架构的openwrt上了，并且带有完整的移植思路。因为害怕发论坛会有风险就选择发群里了，似乎并没有因为我被BAN而消失，有需要的人可以看看。

- 下面是我在群里一些活跃的记录，大家应该能看出来我是喜欢开玩笑的。
<img title="uu移植完整版" src="https://github.com/coolsnowwolf/lede/assets/67340056/4f435370-1de4-494d-96a2-87c430fbe7e9" alt="uu移植完整版"   style="zoom:25%;">
<img title="交流学习以及教新人" src="https://github.com/coolsnowwolf/lede/assets/67340056/28d8dd4c-825f-4329-821f-fb0efdd41b6a" alt="交流学习以及教新人"   style="zoom:25%;">

**Comment by WYC-2020** [comment]
从来就不去tg上面交流，最多在issue活跃几下 看到有能力修的给修修，看到太小白或者重复的issue也会吐槽几句，目前就这样，有的人完全伸手党连搜索都懒得搜

**Reply by blzadx** [reply]
谢谢回复，主要我所知道的lede最大的交流区就是tg群了，有1万多人。
关于小白和伸手党的问题，小白只要他是真感兴趣，真想学知识，我有空都是愿意指点一下的，截图里你也看得到，毕竟圈子总还是需要新鲜血液的。何况谁还不是从一无所知的阶段过来的呢。而且交流群壮大了，总会有人愿意做这些事，有人指点，小白也能更快地入门。
至于伸手党，不理会他们就行了，此处没有回应，他们自然会去别处伸手。因为这些伸手党总是数量众多，有的时候还能给小圈子的宣传起一点作用。我认为只要他们不搞破坏，也没必要排挤他们。倒不如说，有人伸手，至少说明这个圈子还有点热度，等到连伸手党都没了，那这个圈子大概就真没落了。
